hypervisor/arch/x86/assign.c

```diff
@@ -669,7 +670,7 @@ int32_t ptirq_intx_pin_remap(struct acrn_vm *vm, uint8_t virt_pin,
 				 * switch vpin source is needed
 				 */
 				if (virt_pin < NR_LEGACY_PIN) {
-					uint8_t vpin = pic_ioapic_pin_map[virt_pin];
+					uint8_t vpin = get_ioapic_pin_from_pic_pin(virt_pin);
 
```

Comment:
here actually is mapping between pic & ioapic, but usually it's get_pic_pin_from_ioapic_pin.